docs/asciidoc-writers-guide.adoc

```diff
@@ -820,6 +820,9 @@ h|Verbatim |Yes                |Yes      |No     |No         |No           |No
 h|None     |No                 |No       |No     |No         |No           |No     |No
 |===
 
+
+In order to apply normal substitutions to the `citetitle` of a quote block you must surround the attribute value with single quotes.
```

Comment:
I went back to the source and it turns out that any block attribute value (explicit: `key='value'` or positional: `'value'`) is subject to normal substitutions except for the `options` and `title` attributes. The `options` attribute value never gets substituted. The `title` attribute value always receives title substitutions.

Here's the relevant section of the code.

* https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor/attribute_list.rb#L160-L182

There are some edge cases, but we can at least start by documenting as: single quotes mean "apply normal subs".

Aside from this change request, :+1: to the update. Thanks!